src/providers/ipa/ipa_deskprofile_rules_util.c

```diff
@@ -264,7 +264,9 @@ ipa_deskprofile_rules_create_user_dir(
         goto done;
     }
 
-    ret = sss_create_dir(domain_dir, shortname, 0600, uid, gid);
+    old_umask = umask(0077);
```

Comment:
Please add a comment here on the reason why this specific umask is used.
note that the X bit on a directory is needed to be able to travese it, not to create files in it technically (you still need the W flag to be able to create entries in the dentry).
But w/o traverse you can't do much generally, so it is ok to give it.

Reply:
Okay, a comment has been added.